person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(_name, age, _parent_permission)
+    @id = Random.rand(1..1000)
+    @name = 'Unknown'
+    @age = age
+    @parent_permission = true
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
+
+  def can_use_services?
+    age >= 18 || parent_permission
+  end
```

Comment:
- [ optional ] Well done in applying the changes. ill just recommend putting the `can_use_services?` method before the private since its a public method
i.e 
```
  def can_use_services?
    age >= 18 || parent_permission
  end

  private

  def of_age?
    age >= 18
  end
```

Reply:
Thank you @m05e5 for approving.